polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs

```diff
@@ -129,7 +129,6 @@ impl BenchCli {
 		for (index, CliTestConfiguration { objective, mut test_config }) in
 			test_sequence.into_iter().enumerate()
 		{
-			let benchmark_name = format!("{} #{} {}", &self.path, index + 1, objective);
```

Comment:
We still need the benchmark name. Despite removing it from struct, the final output should be the same as before. Please add it to a `println!` that prints the usage. The same goes for printing in regression tests.

Reply:
@AndreiEres Thanks for the review. Please review my changes as requested. Thanks